jetty-start/src/test/java/org/eclipse/jetty/start/fileinits/MavenMetadataTest.java

```diff
@@ -71,7 +71,7 @@ public void testIsExpiredTimestampNextWeek()
     @Test
     public void testIsExpiredTimestampNow()
     {
-        LocalDateTime now = LocalDateTime.now();
+        LocalDateTime now = LocalDateTime.now(ZoneId.of("UTC"));
```

Comment:
This was a flaky test.
I just fixed it here as part of this PR.
It's really unrelated to the purpose of this PR, just happens to be in the same `jetty-start` code tree.